src/main/java/com/google/cloud/tools/maven/DeployAllMojo.java

```diff
@@ -18,16 +18,14 @@
 
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugins.annotations.Execute;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
 
 /**
  * Stage and deploy the application and all configs to Google App Engine standard or flexible
  * environment.
  */
-@Mojo(name = "deployAll")
-@Execute(phase = LifecyclePhase.PACKAGE)
+@Mojo(name = "deployAll", defaultPhase = LifecyclePhase.PACKAGE)
```

Comment:
I think we picked the wrong default when the plugin was made. DeployXYZ should probably be in the DEPLOY phase by default.